exercises/hello-world/canonical-data.json

```diff
@@ -1,23 +1,12 @@
 {
-   "#": [
-     "Language implementations vary on the issue of missing input variables.",
-     "In this set of test cases, the input variable is left out of the definition,",
-     "and the generator would need to handle the language implementation."
-   ],
-   "cases": [
-      {
-         "description": "no name",
-         "expected": "Hello, World!"
-      },
-      {
-         "description": "sample name",
-         "name": "Alice",
-         "expected": "Hello, Alice!"
-      },
-      {
-         "description": "other sample name",
-         "name": "Bob",
-         "expected": "Hello, Bob!"
+  "exercise":"hello-world",
+  "version":"0.1.0",
+  "group":[
+    {
+      "hello":{
+        "description":"Say Hi!",
+        "expected":"Hello, World!"
```

Comment:
There is no standard format for the `canonical-data.json`, so I used the one I proposed [here](https://github.com/exercism/x-common/issues/336#issuecomment-278550730). If by any chance it gets accepted without change, it will be one less file to update. 😄